docs/en/05_Contributing/01_Code.md

```diff
@@ -105,15 +108,16 @@ When contributing a backwards incompatible change, you must raise it against the
 ### The Pull Request Process
 
 Once your pull request is issued, it's not the end of the road. A [core committer](/contributing/core_committers/) will most likely have some questions for you and may ask you to make some changes depending on discussions you have.
-If you've been naughty and not adhered to the coding conventions, expect a few requests to make changes so your code is in-line.
+If you've been naughty and not adhered to the [coding conventions](coding_conventions), 
```

Comment:
I think the link here should be `[coding conventions](/contributing/coding_conventions)`? ;)